src/pages/TableList.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import Tab from '../components/Tab'
+import StatsTable from '../components/StatsTable'
+
+export default class TableList extends Component {
+  constructor(props) {
+    super(props);
+
+    /** mock up table data  */
+    const teams = [
```

Comment:
nice mocks for testing, we should start writing tests (I will set up some testing framework later). But mocks should not be part of production code, if you rebase your branch against master

git rebase master

you will be able to use new functions from data provider that return similar data